.changes/unreleased/alloy-TINY-11549-2024-12-04.yaml

```diff
@@ -0,0 +1,6 @@
+project: alloy
+kind: Improved
+body: Anchor center position now also consider boundaries.
```

Comment:
```suggestion
body: Centering an element now also considers boundaries.
```
??

Reply:
I changed, but I have a doubt now :sweat_smile: , is not that the action of centering weren't consider the boundaries, but that the center position weren't considering the boundaries but only the element.

Should I revert the change fixing the conjugation of the verb?
```
body: Anchor center position now also considers boundaries.
```

Reply:
What about says what the change is but very long. 😊
Anchor center X position was calculated based on the anchor width instead of the anchor width clamped by the bounds width.

Reply:
Alloy changelogs don't matter too much - they're not published to customers.

Or at all, until we get around to rebuilding the open source publishing scripts.

Reply:
changed with: `Anchor center X position was calculated based on the anchor width instead of the anchor width clamped by the bounds width.`